packages/core/js/colors.2020.js

```diff
@@ -1,3 +1,7 @@
+export const colorsWhite = '#FFFFFF'
+
+export const colorsBlack = '#000000'
```

Comment:
Pedantic but I think we should make this an object so it mirrors other color objects maybe???

Reply:
I checked with Brian.  These are the facts that made me keep it this one:

- It's a single value color, no spectrum like the others.
- Nobody says things like "white base"